src/api/spec/models/watched_item_spec.rb

```diff
@@ -0,0 +1,176 @@
+require 'rails_helper'
+
+RSpec.describe WatchedItem do
+  let(:user) { create(:confirmed_user) }
+
+  context 'when watching a project' do
+    let(:project) { create(:project) }
+    let(:watched_item) { create(:watched_item, :for_projects) }
+
+    it 'is valid with valid attributes' do
+      expect(watched_item).to be_valid
+    end
+
+    it 'is not valid when missing a project' do
```

Comment:
Why test this? You expect `ActiveRecord::Associations` to break?

Reply:
done